akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -834,25 +837,38 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
 
   // InboundContext
   override def sendControl(to: Address, message: ControlMessage) =
-    association(to).sendControl(message)
-
-  override def send(message: Any, sender: OptionVal[ActorRef], recipient: RemoteActorRef): Unit = {
-    val cached = recipient.cachedAssociation
+    try {
+      association(to).sendControl(message)
```

Comment:
it might be places where we call the association's `sendControl` directly, so it's better to move the silence to that method